packages/ava/__tests__/bootstrap.ava.ts

```diff
@@ -0,0 +1,39 @@
+import process from 'node:process';
+import {join} from 'node:path';
+import {spawnSync} from 'node:child_process';
+import {existsSync} from 'node:fs';
+import {removeSync} from 'fs-extra';
+import test from 'ava';
+
+const TEST_PROJECT = join(process.cwd(), '../test-near-runner-ava-bootstrap');
+
+test.before(() => {
+  if (existsSync(TEST_PROJECT)) {
+    throw new Error(`${TEST_PROJECT} already exists! Aborting.`);
```

Comment:
Given that this likely creates the folder as a sibling of the main near-runner folder on your system, I wanted to make guard against the extremely unlikely situation that this folder name doesn't conflict with one that's already there.

Reply:
Why not create the directory in the `__tests__` dir?

Reply:
That would work for now, but I once ran into false positives when using that strategy with near-api-js. The issue was caused by missing dependencies in the created project, which were available in the `node_modules` of the main project. Since the test project was a subfolder, node's resolution logic found the dependencies, even though actually using `create-near-app` resulted in missing dependencies for real use-cases.

If we add tests in the future that try to run tests or something, we could run into this same error.